tests/foreman/cli/test_host.py

```diff
@@ -161,17 +162,15 @@ def test_positive_create_with_name(self):
             with self.subTest(name):
                 host = entities.Host()
                 host.create_missing()
-                result = Host.create({
+                result = make_host({
                     u'architecture-id': host.architecture.id,
                     u'domain-id': host.domain.id,
                     u'environment-id': host.environment.id,
-                    # pylint:disable=no-member
```

Comment:
No, both `make_host` and `Host.create` treat it like dict the same way, but it's not the case, pylint just analyzes all expressions and in case it's call to some class instance member - it tries to ensure the member is there. The issue is `host` doesn't have `location` and `organization` members defined from the very beginning, they're dynamically added later, that's why it's complaining and why it was shut up :)
Regarding your question - i don't really mind, pylint is static analyzer and python is dynamic language, such false alarms are very common so i ignore them anyway and afaik we're not trying to achive 100% pylint score, i was just curious what was a reason for this change.